### DISCUSSION TITLE
Sonoff and Ravencore V1

### DISCUSSION THREAD

**Discussion by Przylepzwm** [RavenSystem/esp-homekit-devices#2037] [General]
Hallo!
I have been using for three years:
-Sonoff mini (HAA V0.8.6)
-Sonoff basic (HAA V1.2)
Since they are used for light control and I have difficult access I have not upgraded to V2. So far I have not had any problem with them.
Yesterday I was forced to disconnect all cables from my router and Apple TV(4th gen) and since then all the above devices are unreachable in HK (they are visible on the network).
I have performed:
-Restarting all accessory centers
-Restarting the router
-Disabling/enabling the power supply on the Sonoff

Any advice on how to get them working again ?

*Sonoff mini works changing the state of the output from the switch and the built-in button.
**I am able to start WifiMode
*** I recently changed the HomeKit architecture, it is possible that since then the first time Apple TV and Router was without power

**Comment by xuncia** [comment]
Hello,
same here. 
1 week ago 5 sonoff switches, that were perfectly working in HomeKit, they stopped working. 
I upgraded the hk architecture too but it was more than 1 week ago....so I don't understand. Manually they are working fine and also wifi mode is working. Any idea is really appreciated. Thank you.

**Reply by Przylepzwm** [reply]
It is working after reset HomeKit ID in setup mode and add as new device to HomeKit. Unfortunately all automations to rewrite...
